episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+){alt="A screenshot of Napari's picker button" height='30px'} icon do?
+- What does the 'contour' setting control?
+- What does the 'n edit dim' setting control?
+- What does the 'preserve labels' setting control?
+
+Remember that you can hover over buttons to see a helpful popup with some more 
+information. You can also look in [Napari's documentation
+](https://napari.org/stable/howtos/layers/labels.html) on `Labels` layers.
+
+Note that if you make a mistake you can use <kbd>Ctrl</kbd> + <kbd>Z</kbd> to 
+undo your last action. If you need to re-do it use 
+<kbd>Shift</kbd> + <kbd>Ctrl</kbd> + <kbd>Z</kbd>.
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+###  ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/fill.svg
+){alt="A screenshot of Napari's fill button" height='30px'} icon
+
+This icon activates the 'fill bucket'. It will fill whatever label you click on 
+with the currently active label (as shown next to 'label:' in the layer 
+controls). For example, if your active label is 2, then clicking on a nucleus 
+with label 1 will change it to 2. You can see more details about the fill bucket 
+in [Napari's documentation
+](https://napari.org/stable/howtos/layers/labels.html#using-the-fill-bucket)
+
+### ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/picker.svg
+){alt="A screenshot of Napari's picker button" height='30px'} icon
+
+This icon activates the 'colour picker'. This allows you to click on a pixel in 
+the viewer and immediately select the corresponding label. This is especially 
+useful when you have hundreds of labels, which can make them difficult to keep 
+track of with the 'label:' option only.
+
+### Contour
+
+The contour setting controls how labels are displayed. By default (when 0), 
+labelled pixels are displayed as solid colours. Increasing the contour value to 
+one or higher, will change this to show the outline of labelled regions instead. 
+Higher values will result in a thicker outline. This can be useful if you don't 
+want to obscure the underlying image while viewing your segmentation.
+
+### n edit dim
+
+This controls the number of dimensions that are used for editing labels. By 
+default, this is 2 so all painting and editing of the `Labels` layer will only 
+occur in 2D. If you want to paint/edit in 3D you can increase this to 3. Now 
+painting will not only affect the currently visible image slice, but slices 
+above and below it also. Be careful with the size of your brush when using this 
+option!
+
+### preserve labels
+
+Activating 'preserve labels' will ensure that any painting/erasing you do will 
+only affect your currently active label. This is especially useful if you need 
+to paint very close to existing labels without affecting them.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Manual vs automated?
```

Comment:
I think we should flesh this out a bit more. Manual in some cases has been the training set from which automated methods are based. This is particularly the case in terms of  tricky situations when the boundaries are subtle and expert segmentations help guided automated ones.  However, manual segmentations are also more likely to have biases and increased variability and not be as repeatable as automated scan. And as you already point out, they can be slow and arduous (which again will lead to more bias and increased variability!) I think your points both about manual review and cleanup are important, but don't forget manual initialisation leading to semi-automated approaches (i.e. providing a starting seed/guess), which is also a good technique.